client/js/editmode.js

```diff
@@ -96,6 +96,31 @@ function editClick(widget) {
   showOverlay('editOverlay');
 }
 
+function generateEmptyDeckWidget(id, x, y) {
+  const widgets = [
+    { type:'holder', id, x, y, dropTarget: { type: 'card' } },
+    { type:'pile', id: id+'P', parent: id, width:103, height:160 },
```

Comment:
You shouldn't add an empty pile. It will be created automatically when adding cards to the deck (at least it should).

In general I feel like this should reuse the code instead of copying it but that can be done later.

Reply:
Got it, will remove and test.